### DISCUSSION TITLE
Location editing copy paste coordinates

### DISCUSSION THREAD

**Discussion by iqapps** [immich-app/immich#8380] [Feature Request]
### The feature

Thanks for being able to edit photos location.
I have a large pool og photos I need to edit location for, and many of these are taken at the same position, my home.
It would be nice to be able to pick location for one, get the cps coordinates, and paste that copy for the others.
I know I can select many photos, but I also need to edit text and time for most of them, so I edit single photos mostly.
Anyway, it would just be nice to be able to pick one, copy the location, and then use that location for a set of other photos to be located at the same spot.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by sannidhyaroy** [comment]
This would be a life saver as I'm stuck in the same boat...

**Comment by bo0tzz** [comment]
#11291
